manifests/consolelink/consolelink.yaml

```diff
@@ -0,0 +1,13 @@
+  href: <dashboard-url>
+  location: ApplicationMenu
+  text: Open Data Hub
```

Comment:
2 values "href" and "text":
1. i saw you have the overlay of rhoai, to replace text, which should cover for both ODH and rhoai already
2. i dont know if the plan is to get consolelink for both ODH and downstream assume this is the reason changed from <rhods-dashboard-url> to <dashboard-url> for "href"
then operator need to do some updates for this part. just need to confirm, you have decided to use dashboard-url as the placeholder here